test/models/erratum_test.rb

```diff
@@ -131,9 +131,9 @@ def setup
     end
 
     def test_hosts_available
-      assert_includes @security.hosts_available, @host.content_facet
-      refute_includes @security.hosts_available, @host_without_errata
-      refute_includes @bugfix.hosts_available, @host_without_errata
+      assert_includes @security.hosts_available(::Organization.first.id), @host.content_facet
+      refute_includes @security.hosts_available(::Organization.first.id), @host_without_errata
+      refute_includes @bugfix.hosts_available(::Organization.first.id), @host_without_errata
```

Comment:
I think one of the tests is failing because here you'd want to pass in @host.organization.id (rather than Organization.first.id.

Could you test for both, i want to make sure that the condition where id is passed in and when its not passed in are both covered.